client/src/components/AddVideo.js

```diff
@@ -0,0 +1,97 @@
+import React,{ useState } from "react";
+
+const dateFormat = (date) => {
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1;
+  const day = date.getDate();
+  return `${day}-${month}-${year}`;
+};
+
+const timeFormat = (date) => {
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const seconds = date.getSeconds();
+  return `${hours}-${minutes}-${seconds}`;
+};
+const AddVideo = (prop) => {
+  const [title, settitle] = useState("");
```

Comment:
nice job splitting AddVideo into its own component! Only a tiny thing, but try remember to use camelCase (setTitle) - know you've used it elsewhere which is good

Reply:
done